zomes/hc_zome_trust_graph/src/trust_graph.rs

```diff
@@ -0,0 +1,91 @@
+use hdk::prelude::*;
+use hc_zome_trust_atom::*;
+use hc_zome_trust_atom::TrustAtom;
+use std::collections::hash_map;
+
+#[derive(Debug, Clone, Default, Serialize, Deserialize, SerializedBytes)]
+struct TrustGraph {
+  trust_atoms: BTreemap<u64, TrustAtom>, //listed by id
+  rollup: TrustAtom, // algorithm from agents you placed TA for given filter
+}
+
+impl TrustGraph {
+    fn create(agents: Vec<EntryHash>, tag_filter: Option<LinkTag>) -> ExternResult<Self> { // TODO: construct agent list for input. Enum?
```

Comment:
In my vision we don't want to pass a list of agents, we want to roll up the atoms created by "me" -- and those of the agents that my atoms point to.

Reply:
I suggest we rename this method to `create_rollup_atoms`